README.md

```diff
@@ -1,42 +1,84 @@
   <!-- badges: start -->
-  [![CircleCI build status](https://circleci.com/gh/PolicyAnalysisInc/heRoMod.svg?style=svg)](https://circleci.com/gh/PolicyAnalysisInc/heRoMod)
- [![Coverage status](https://codecov.io/gh/PolicyAnalysisInc/heRoMod/branch/master/graph/badge.svg)](https://codecov.io/github/PolicyAnalysisInc/heRoMod?branch=master)
+
+[![CircleCI build status](https://circleci.com/gh/PolicyAnalysisInc/heRoMod.svg?style=svg)](https://circleci.com/gh/PolicyAnalysisInc/heRoMod)
+[![Coverage status](https://codecov.io/gh/PolicyAnalysisInc/heRoMod/branch/master/graph/badge.svg)](https://codecov.io/github/PolicyAnalysisInc/heRoMod?branch=master)
+
   <!-- badges: end -->
 
 # heRomod - A Package for Health Economic Modeling
 
-The heRomod package allows users to develop and run cost-effectiveness models in R and is the calculation engine used in the heRo web-based modeling platform.  A fork of the HEEMOD package.
+The heRomod package allows users to develop and run cost-effectiveness models in R and is the calculation engine used in the heRo web-based modeling platform. A fork of the HEEMOD package.
 
 You can install the latest released version from github with:
 
-```r
+## Developing Locally
+
+If you want to run the package locally from source
+
+1. Install R (>= 3.4.0) for your platform.
+    1. For Windows users, it is recommended to check Save version number in registry during installation so that the R extension can find the R executable automatically.
+    1. For mac users, use brew to install R and dependencies in terminal
+        1. `brew install R`
+        1. `brew install libgit2 harfbuzz fribidi freetype2 libpng libtiff libjpeg pandoc`
+1. Install languageserver in R.
+1. Open an R terminal
+
+```R
+install.packages("languageserver")
+install.packages("usethis")
+install.packages("pkgdown")
+install.packages("devtools")
+```
+
+1. Install the R extension for Visual Studio Code.
+1. Test the build command
+    1. `devtools::build(vignettes=T)`
+    1. `devtools::install(dependencies=F)`
+1. Run the tests (note - this will take significant time, especially running the heRo model)
+    1. `devtools::test()`
+
+## Running the heroMod package locally
+
+1. Install the Github package manager and Hero remotes
+
+```R
+
 if(!require(remotes)) {
   install.packages('remotes')
 }
 Sys.setenv("R_REMOTES_NO_ERRORS_FROM_WARNINGS" = "true")
+remotes::install_github("PolicyAnalysisInc/herotools")
+remotes::install_github("PolicyAnalysisInc/herosurv")
+```
+
+1. Install heromod
+1. If prompted you can install "All"
+
+```R
```

Comment:
This finished with `* DONE (heRomod)` but it threw plenty of errors during the execution